android-stellar-sdk/proguard-rules.pro

```diff
@@ -12,6 +12,14 @@
 # Gson specific classes
 -dontwarn sun.misc.**
 
+# Application classes that will be serialized/deserialized over Gson
+-keep class org.stellar.sdk.responses.** { <fields>; }
```

Comment:
Does it keep inner classes also?

Reply:
actually, I've noticed in tests that's the second new rule that does the trick (i.e. R8 is the blame) so I'm not sure,  I kept also this rule as to follow gson instructions (make sense to tell ProGuard to keep data classes fields)

Reply:
Cool, Thanks 👍🏻